demo/vagrant/horizontal-app-scaling/jobs/autoscaler.nomad

```diff
@@ -96,15 +99,12 @@ strategy "target-value" {
 
       config {
         image = "grafana/promtail:1.5.0"
-
+        network_mode = "host"
```

Comment:
What is the reason the network mode param was added? I don't believe we need it.